app/models/user.rb

```diff
@@ -7,4 +8,27 @@ class User < ActiveRecord::Base
                     uniqueness: { case_sensitive: false }
   has_secure_password
   validates :password, presence: true, length: { minimum: 6 }
+
+  def User.digest(string)
+    cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST :
+                                                  BCrypt::Engine.cost
+    BCrypt::Password.create(string, cost: cost)
+  end
+
+  def User.new_token
```

Comment:
__[rubocop]__
* Use `self.new_token` instead of `User.new_token`. [:ref](https://github.com/bbatsov/ruby-style-guide#def-self-class-methods)